src/py_opencc.cpp

```diff
@@ -0,0 +1,50 @@
+#include <pybind11/pybind11.h>
+#include "opencc.h"
+
+class PYOpenCC {
+private:
+    opencc_t converter;
+
+public:
+    std::string config;
+
+    PYOpenCC(const char* configFileName) {
+        config = configFileName;
+        converter = opencc_open(configFileName);
```

Comment:
Why don't we depend on the C++ SimpleConverter API?

Reply:
Good point! Saves rewriting of code.

Reply:
Done 32a1ec7 f52dad0